packages/styleguide/src/components/Preview/Preview.test.js

```diff
@@ -8,4 +8,19 @@ describe('rendering', () => {
     const { getByText } = render(<Preview>Preview</Preview>);
     expect(getByText('Preview')).toBeInTheDocument();
   });
+
+  it('HTML preview renders more than 1 child', () => {
+    const { getByText, getByTestId } = render(
+      <Preview>
+        <p>Paragraph 1</p>
+        <p>Paragraph 2</p>
+      </Preview>
+    );
+
+    fireEvent.click(getByText('SHOW CODE'));
+
+    fireEvent.click(getByText('HTML'));
+
+    expect(getByTestId('html')).toBeInTheDocument();
```

Comment:
Ked mam test, ktory checkuje ze ci sa zrenderuje viac ako 1 child, tak by som pocital kolko sa ich zrenderovalo. Napr aj tak, ze si das test-id do renderu tu.

```
<Preview>
  <p data-testid="p1">Paragraph 1</p>
  <p data-testid="p1">Paragraph 2</p>
</Preview>
```

Reply:
no lenze toto nebude fungovat kedze ten vystup je nieco taketo:
```html
<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>svg</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>icon icon--heart<span class="token punctuation">"</span></span> <span class="token attr-name">role</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>presentation<span class="token punctuation">"</span></span> <span class="token attr-name">aria-hidden</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>true<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
```

Reply:
No ale to data-testid tam niekde bude.

Reply:
Aha rozumiem, zrejme nebude.

Stale vies selectovat cez text content cize vies hladat napr. `Paragraph 1` a `Paragraph 2`